README.md

```diff
@@ -15,7 +15,9 @@ Instructions to install on [bosh-lite](https://github.com/cloudfoundry/bosh-lite
 bosh target BOSH_DIRECTOR_HOST
 git clone https://github.com/cloudfoundry-incubator/app-autoscaler-release
 cd app-autoscaler-release
+source .envrc
```

Comment:
If you plan to use envrc then we should expect people to use direnv.
If we need the GOPATH set properly for sync-package-spec then hardcode it in there.